src/sdk/search/Provider.tsx

```diff
@@ -1,12 +1,12 @@
-/* eslint-disable @typescript-eslint/restrict-plus-operands */
-import React, { createContext, useMemo, useState } from 'react'
+/* eslint-disable react/prop-types */
```

Comment:
Why is this necessary? 🤔

Reply:
I have no idea. My eslint is complaining about it

Reply:
That's weird, it might be something wrong in our ESLint setup then. The docs for this specific rule is this one: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md.

Since we're using TypeScript, and typing the `SearchProvider` props, this error should not be happening

Reply:
nice. I'll remove it then

Reply:
I guess this could stay here! I would just add a comment explaining why we need it